app/assets/javascripts/find-data.js

```diff
@@ -0,0 +1,13 @@
+function sortDatasets() {
```

Comment:
This file's name is confusing. And there are no comments to explain what it does.